setup.py

```diff
@@ -0,0 +1,39 @@
+
+# -*- coding: utf-8 -*-
+
+# DO NOT EDIT THIS FILE!
+# This file has been autogenerated by dephell <3
+# https://github.com/dephell/dephell
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+import os.path
+
+readme = ''
+here = os.path.abspath(os.path.dirname(__file__))
+readme_path = os.path.join(here, 'README.rst')
+if os.path.exists(readme_path):
+    with open(readme_path, 'rb') as stream:
+        readme = stream.read().decode('utf8')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found indentation with spaces instead of tabs](https://app.codacy.com/app/pwwang/pyparam/pullRequest?prid=4632439)